syntax = "proto3";
package role;





/*
RoleChangePermission请求
*/
message RoleChangePermissionRequest {
    /*
    添加权限id列表
    */
    repeated string add_permissions = 1 ;
    /*
    删除权限id列表
    */
    repeated string delete_permissions = 2 ;
    /*
    角色id
    */
    string id = 3 ;
}

/*
RoleChangePermission返回
*/
message RoleChangePermissionResponse {
    /*
    角色成功修改的权限数量
    */
    int32 count = 1 ;
}

/*
RoleChangePermissionApi返回
*/
message RoleChangePermissionResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    role.RoleChangePermissionResponse data = 4 ;
}

